tests/InstrEngineTests/TestScripts/AddExceptionHandler_ReturnMultiMethodTest.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<InstrumentationTestScript>
+  <InstrumentMethod>
+    <ModuleName>AddExceptionHandlerTests_Release_x64.exe</ModuleName>
+    <MethodName>ReturnMultiMethodTest</MethodName>
+    <AddExceptionHandler>True</AddExceptionHandler>
+    <MakeSingleRet />
```

Comment:
><MakeSingleRet /> [](start = 4, length = 17)

Just for clarity, can you put this tag above the exception handler, and add a comment saying that exception instrumentation does not handle multi returns?

Reply:
Done in this commit: https://github.com/robertpi/CLRInstrumentationEngine/commit/d01b06ef411a69720c80d5d59f4f997120d2c092

I renamed the files, so don't think a comment is necessary.